pumpkindb_engine/src/script/mod.rs

```diff
@@ -387,7 +387,8 @@ impl<'a, T: Dispatcher<'a>> Scheduler<'a, T> {
                 // Otherwise, if there's more than one Env left,
                 // pick the next Env to schedule randomly
                 if len > 1 {
-                    let index: usize = rng.gen_range(1, len);
+                    // TODO: are we supposed to swap 0 with index instead of len-1?
+                    let _index: usize = rng.gen_range(1, len);
```

Comment:
It's not for testing, the idea was to swap with a random Env. I just sent in a PR on this https://github.com/PumpkinDB/PumpkinDB/pull/330